app/_src/gateway/kong-enterprise/secrets-management/advanced-usage.md

```diff
@@ -128,8 +128,8 @@ Most of the vaults also support secret rotation by using TTLs:
 
 Parameter | Field name | Description
 ----------|------------|------------
-`vaults.config.ttl` | **TTL** | Time-to-live (in seconds) of a secret from the vault when cached by this node.
+`vaults.config.ttl` | **TTL** | Time-to-live (in seconds) of a secret from the vault when cached.
 `vaults.config.neg_ttl` | **Negative TTL** | Time-to-live (in seconds) of a vault miss (no secret).
```

Comment:
I believe the `neg_ttl` description could be expanded more to make it clearer.
For example by adding something like the below explanation (if the below explanation is correct),
```
negatively cached secrets will remain valid until neg_ttl is reached, after which Kong will attempt to refresh the secret again
```

Reply:
added this to the description in all fields.